src/printer.js

```diff
@@ -181,6 +181,20 @@ function genericPrintNoParens(path, options, print, args) {
     case "EmptyStatement":
       return "";
     case "ExpressionStatement":
+      // Detect Flow-parsed directives
+      if (n.directive) {
+        return concat([
+          nodeStr(
+            {
```

Comment:
Oops, sorry about that! I thought this was good to go, but apparently not. I'll make sure to let you merge the next attempt :)